src/material/_theming.scss

```diff
@@ -0,0 +1,8 @@
+// Forwards all public API mixins so they can be imported from a single entry point.
```

Comment:
Do we need a `theming.import.scss` in the root?

Reply:
I'm not sure. This file is primarily tailored towards `@import` users since all the symbols will be prefixed. I don't know whether we'd want something like this for `@use` users, because there's a high chance for collisions (e.g. all themes have a mixin called `color`). It might be better to recommend using deep imports together with `@use`.

Reply:
We can probably tackle this in a follow-up. I think for `@use` API, we'd want to have e.g.
```scss
@use '~@angular/material' as mat;
@include mat.button-theme(...);
```
whereas for `@import` we'd obviously need to keep the compat